guardrails/cli/start.py

```diff
@@ -39,7 +39,7 @@ def start(
 ):
     logger.debug("Checking for prerequisites...")
     if not api_is_installed():
-        package_name = 'guardrails-api>="^0.0.0a0"'
+        package_name = 'guardrails-api>=0.0.0a0'
```

Comment:
Make the linter happy

```suggestion
        package_name = "guardrails-api>=0.0.0a0"
```